src/hotspot/share/opto/superword.cpp

```diff
@@ -2258,27 +2308,39 @@ void SuperWord::schedule() {
     }
 #endif
     _packset.clear();
-    return;
+    return false;
   }
 
-#ifndef PRODUCT
-  if (is_trace_superword_info()) {
-    tty->print_cr("SuperWord::schedule: memops_schedule:");
-    memops_schedule.dump();
-  }
-#endif
```

Comment:
Note: debug printing moved to `SuperWord::apply_memops_reordering_with_schedule`